src/Goerli-DssSpell.t.base.sol

```diff
@@ -549,15 +549,15 @@ contract GoerliDssSpellTestBase is Config, DSTest, DSMath {
             }
             uint256 normalizedTestDust = values.collaterals[ilk].dust * RAD;
             assertEq(dust, normalizedTestDust, concat("TestError/vat-dust-", ilk));
-            assertTrue((dust >= RAD && dust < 100 * THOUSAND * RAD) || dust == 0, concat("TestError/vat-dust-range-", ilk)); // eq 0 or gt eq 1 and lt 100k
+            assertTrue((dust >= RAD && dust <= 100 * THOUSAND * RAD) || dust == 0, concat("TestError/vat-dust-range-", ilk)); // eq 0 or gt eq 1 and lte 100k
             }
 
             {
             (address pip, uint256 mat) = spotter.ilks(ilk);
             if (pip != address(0)) {
                 // Convert BP to system expected value
                 uint256 normalizedTestMat = (values.collaterals[ilk].mat * 10**23);
-                if ( values.collaterals[ilk].lerp ) {
+                if ( values.collaterals[ilk].lerp || ilk == bytes32("RENBTC-A")) {
```

Comment:
It looks like this is the only place where we use `lerp`

It may be more approprate to rename the struct value `offboarding` and set it to `true` for `RENBTC-A`, then we shouldn't need to make a carveout here.

Reply:
This is much cleaner, thanks.